packages/core/src/util/txInspector.ts

```diff
@@ -24,35 +28,96 @@ export type StakeKeyRegistrationInspection = StakeAddressCertificate[];
 export type WithdrawalInspection = Lovelace;
 
 // Inspector types
+interface SentInspectorArgs {
+  addresses?: Address[];
```

Comment:
Is there a reason to have these args as nullable?

Reply:
Makes sense, just make sure different combinations are covered by tests

Reply:
Question, does it make sense if I remove the `stakeKeyHashes` argument and ask only for `rewardAccounts`? And then do something like this:

```ts
const stakeKeyHashes = rewardAccounts?.map((account) => Ed25519KeyHash.fromRewardAccount(account));
```

I think it'd make it easier to use as with the wallet you can just use `wallet.delegation.rewardAccounts$`

Reply:
Agree, good idea :+1: